[INFO -- 2018-11-13 17:48:01]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-13 17:48:01]
Initialization strings:
[INFO -- 2018-11-13 17:48:01]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-13 17:48:01]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-11-13 17:48:01]
initSignal:0
[INFO -- 2018-11-13 17:48:01]
endSignal:5
[INFO -- 2018-11-13 17:48:01]
faultModel:1
[EXCEPTION -- 2018-11-13 17:48:04]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-11-13 17:48:04]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-11-13 17:48:05]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff6f0f897 in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7ffff703cb9a "%s\n") at ../sysdeps/posix/libc_fatal.c:181
#3  0x00007ffff6f1690a in malloc_printerr (str=str@entry=0x7ffff703ad88 "free(): invalid pointer") at malloc.c:5350
#4  0x00007ffff6f1de1c in _int_free (have_lock=0, p=0x5555557723b1, av=0x7ffff7271c40 <main_arena>) at malloc.c:4157
#5  __GI___libc_free (mem=0x5555557723c1) at malloc.c:3124
#6  0x0000555555558018 in __gnu_cxx::new_allocator<double>::deallocate (this=0x7fffffffd720, __p=0x5555557723c1) at /usr/include/c++/7/ext/new_allocator.h:125
#7  0x0000555555557add in std::allocator_traits<std::allocator<double> >::deallocate (__a=..., __p=0x5555557723c1, __n=18446744073709551602) at /usr/include/c++/7/bits/alloc_traits.h:462
#8  0x0000555555557230 in std::_Vector_base<double, std::allocator<double> >::_M_deallocate (this=0x7fffffffd720, __p=0x5555557723c1, __n=18446744073709551602) at /usr/include/c++/7/bits/stl_vector.h:180
#9  0x0000555555556eed in std::_Vector_base<double, std::allocator<double> >::~_Vector_base (this=0x7fffffffd720, __in_chrg=<optimized out>) at /usr/include/c++/7/bits/stl_vector.h:162
#10 0x0000555555556919 in std::vector<double, std::allocator<double> >::~vector (this=0x7fffffffd720, __in_chrg=<optimized out>) at /usr/include/c++/7/bits/stl_vector.h:435
#11 0x000055555555709c in std::vector<double, std::allocator<double> >::_M_move_assign (this=0x55555575b1a0 <T2>, __x=...) at /usr/include/c++/7/bits/stl_vector.h:1531
#12 0x0000555555556971 in std::vector<double, std::allocator<double> >::operator= (this=0x55555575b1a0 <T2>, __x=...) at /usr/include/c++/7/bits/stl_vector.h:466
#13 0x0000555555555dc8 in train (iter1=45000, iter2=45000, N=4, alpha1=2.2222199999999999e-05, alpha2=2.2222199999999999e-05, stopsWhenStable=true) at lreg.cpp:186
#14 0x0000555555556467 in main (argc=3, argv=0x7fffffffde58) at lreg.cpp:253

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping value of field: __val
Fliping array at pos: 6
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Double bit-flip
base address to flip value: 0x7fffffffd260
address max offset: 8
Memory content after  bitflip:0x000x030x000x000x000x000x000x00
frame name: __GI_raise
symbol name: set
symbol filename: ../sysdeps/unix/sysv/linux/raise.c
symbol line: 39
value: {__val = {0, 0, 0, 0, 0, 0, 768, 0, 0, 0, 140737337556952, 140737336153733, 0, 0, 13890566350054621184, 140737336112886}}
value address: 0x7fffffffd230
Type: The type is a structure.
Type sizeof: 128
Field name: __val
Field Type: The type is an array.
Field Type sizeof: 128

[INFO -- 2018-11-13 17:48:05]
Fault Injection Successful after 4.324317216873169s
[INFO -- 2018-11-13 17:48:05]
event type: exit
[EXCEPTION -- 2018-11-13 17:48:05]
exit code: no exit code available
